game/src/common.scss

```diff
@@ -69,4 +70,12 @@ $dark-background: #222222;
 $dark-text-color:  #e0e0e0;
 $dark-good: #466900; 
 $dark-bad: #d87777;
-$dark-input: #414141;
\ No newline at end of file
+$dark-input: #414141;
+
+@mixin not-selectable(){
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
```

Comment:
Move up to other mixins.

Reply:
I couldn't understand your comment. Please explain it clearly

Reply:
Further up in the same file are other mixins. Please move this mixin up there, rather than have it placed next to the colours.